core/app/helpers/mno_enterprise/image_helper.rb

```diff
@@ -32,5 +34,27 @@ def engine_image_path(file_name)
     def fit_image
       'max-width: 150px; max-height: 150px;'
     end
+
+    def main_logo_white_bg_dimensions
+      return FastImage.size(main_logo_white_bg_path(true))
+    end
+
+    # Returns dimensions for the main_logo_white_bg, with width < 150px && height < 150px
+    # while keeping the same aspect ratio
+    def main_logo_white_bg_size_to_fit
+      dimensions = main_logo_white_bg_dimensions
+      original_width = dimensions[0]
+      original_height = dimensions[1]
+      return dimensions if original_width < 150 && original_height < 150
+
+      if original_width > original_height
+        width = [original_width, 150].min
+        height = width * original_height / original_width
+      else
+        height = [original_height, 150].min
+        width = height * original_width / original_height
+      end
+      [width, height]
```

Comment:
Would it be better to return a hash? so you can use the return value straight away?